brainscore/benchmarks/zhu2019.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from numpy import ndarray
+from numpy.random import RandomState
+
+import brainscore
+from brainio.assemblies import DataAssembly
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.benchmarks.screen import place_on_screen
+
+from brainscore.metrics import Score
+from brainscore.metrics.accuracy import Accuracy
+from brainscore.model_interface import BrainModel
+from brainscore.utils import LazyLoad
+from brainscore.metrics.error_consistency import ErrorConsistency
+
+BIBTEX = """@article{zhu2019robustness,
+            title={Robustness of object recognition under extreme occlusion in humans and computational models},
+            author={Zhu, Hongru and Tang, Peng and Park, Jeongho and Park, Soojin and Yuille, Alan},
+            journal={arXiv preprint arXiv:1905.04598},
+            year={2019}
+        }"""
+
+DATASETS = ['extreme_occlusion']
+
+# create functions so that users can import individual benchmarks as e.g. Zhu2019RDM
+for dataset in DATASETS:
+    # behavioral benchmark
+    identifier = f"Zhu2019{dataset.replace('-', '')}Accuracy"
+    globals()[identifier] = lambda dataset=dataset: _Zhu2019Accuracy(dataset)
+
+    # engineering benchmark
+    identifier = f"Zhu2019{dataset.replace('-', '')}AccuracyEngineering"
+    globals()[identifier] = lambda dataset=dataset: _Zhu2019Accuracy_Engineering(dataset)
+
+
+class _Zhu2019Accuracy(BenchmarkBase):
+    # behavioral benchmark: compares model: average humans
+    def __init__(self, dataset):
+        self._assembly = LazyLoad(lambda: load_assembly(dataset))
+        self._fitting_stimuli = brainscore.get_stimulus_set('yuille.Zhu2019_extreme_occlusion')
+        self._stimulus_set = LazyLoad(lambda: load_assembly(dataset).stimulus_set)
+        self._visual_degrees = 8
+        self._number_of_trials = 1
+
+        self._metric = Accuracy()
+
+        super(_Zhu2019Accuracy, self).__init__(
+            identifier=f'yuille.Zhu2019_{dataset}-accuracy',
+            parent='yuille.Zhu2019',
+            ceiling_func=lambda: Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation']),
```

Comment:
this should compute a ceiling

```suggestion
            ceiling_func=lambda: calculate_ceiling(self._assembly, self._metric),
```